api/sse/tasks.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+import requests
+from django.conf import settings
+
+from task_processor.decorators import register_task_handler
+
+
+@register_task_handler()
+def send_environment_update_messages(environment_keys: List[str]):
+    if not settings.SSE_SERVER_BASE_URL:
+        return
+
+    for environment_key in environment_keys:
```

Comment:
Perhaps a slight improvement here would be to create another task that is `send_environment_update_message(environment_key: str)` then this function can call that directly (probably without delaying it) but then consumers can call one or the other rather than unnecessarily building a list of one environment key?

Reply:
I don't think that's needed with the current code?

Reply:
Perhaps not, although the logic here: https://github.com/Flagsmith/flagsmith/pull/1476/files#diff-5790db124fc097514cade0bfbcb7898a53274909400a6a5c7859a8fdc5c81e9fR137 may be slightly neater with this approach